client/src/components/DataFiles/DataFilesListing/DataFilesListing.test.js

```diff
@@ -243,6 +245,34 @@ describe('DataFilesListing', () => {
     fireEvent.change(dropdownSelector, { target: { value: 'Folders' } });
     expect(getAllByTestId('file-listing-item').length).toBe(4);
     fireEvent.change(dropdownSelector, { target: { value: 'All Types' } });
-    expect(getAllByTestId('file-listing-item').length).toBe(filesFixture.listing.FilesListing.length);
-  })
+    expect(getAllByTestId('file-listing-item').length).toBe(
+      filesFixture.listing.FilesListing.length
+    );
+  });
+
+  it('not rendering Shared Workspaces component', () => {
```

Comment:
We typically try to write the `it` functions as readable statements
```suggestion
  it('does not render the Shared Workspaces component', () => {
```